src/convertToObject.js

```diff
@@ -6,7 +6,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const result = {};
+  let styleRules = sourceString.split(';');
+
+  styleRules = styleRules.filter((rule) => rule.trim().length);
+
+  for (const line of styleRules) {
+    const [rule, value] = line.split(':');
 
+    result[rule.trim()] = value.trim();
+  }
```

Comment:
[CODE PRACTICE] - Avoid using loops in this task, use iteration methods instead. You could use `forEach` to iterate over `styleRules` after filtering, or better yet, use `reduce` to transform the `styleRules` array directly into an object.